src/components/Loader.spec.js

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+import { render, waitFor, screen } from "@testing-library/react";
+import "@testing-library/jest-dom";
+import Loader from "./Loader";
+
+jest.mock("react-router-dom", () => {
```

Comment:
why are we doing this mock

Reply:
It was needed in screens, but not needed in components. So removed it.